src/player.h

```diff
@@ -653,6 +653,8 @@ class player : public Character
         bool immune_to( body_part bp, damage_unit dam ) const;
         /** Calls Creature::deal_damage and handles damaged effects (waking up, etc.) */
         dealt_damage_instance deal_damage(Creature *source, body_part bp, const damage_instance &d) override;
+        /** Reduce healing effect intensity, return initial intensity of the effect */
+        reduce_healing_effect( const efftype_id &eff_id, int remove_med, body_part hurt );
```

Comment:
Return type `int` is missing for `reduce_healing_effect` function in `player.h`.